A/ArcadeLearningEnvironment/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ArcadeLearningEnvironment"
+version = v"0.1.0"
```

Comment:
We prefer to use the same version as the upstream package, so in this case it would be
```suggestion
version = v"0.6.1"
```